src/main/java/seedu/club/logic/commands/AddPollCommand.java

```diff
@@ -50,6 +50,8 @@ public AddPollCommand(Poll poll) {
     public CommandResult executeUndoableCommand() throws CommandException {
         requireNonNull(model);
         try {
+            requireToSignUp();
+            requireToLogIn();
```

Comment:
Require shoulnt be in try catch. Should be bfr. This applies to other commands as well

Reply:
Oh! OK I’ll fix it